Gemfile

```diff
@@ -36,65 +36,65 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+#My gems
 gem 'awesome_print'
 gem 'dotenv'
 gem 'dotenv-rails'
 gem 'httparty'
 gem 'nokogiri'
 gem 'puma_worker_killer'
-gem "rubycritic"
+gem 'rack-canonical-host'
 gem 'sassc-rails'
 gem 'sdoc',                 group: :doc
 gem 'slim'
 gem 'slim-rails'
 gem 'therubyracer',         platforms: :ruby
-gem 'rack-canonical-host'
+
 # assets
-gem "font-awesome-sass"
-gem "font-awesome-rails"
+gem 'font-awesome-sass'
+gem 'font-awesome-rails'
 gem 'jquery-rails'
 gem 'jquery-ui-rails'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug',         platforms: [:mri,  :mingw,  :x64_mingw]
-  gem 'hirb'
-  gem 'guard-cucumber'
-  gem 'pronto'
-  gem 'pronto-rubocop'
-  gem 'pronto-slim'
-  gem "slim_lint"
   gem 'coveralls',      require: false
+  gem 'flog'
+  gem 'guard-cucumber'
+  gem 'hirb'
   gem 'inch', require: false
-  gem "overcommit"
+  gem 'overcommit'
+  gem 'rainbow',        '~> 2.2.2'
+  gem 'pronto',         '>= 0.9.5'
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `pronto` should appear before `rainbow`.